osu.Game/Screens/Import/FileImportScreen.cs

```diff
@@ -0,0 +1,320 @@
+                                                                    {
+                                                                        Anchor = Anchor.BottomCentre,
+                                                                        Origin = Anchor.BottomCentre,
+                                                                        RelativeSizeAxes = Axes.X,
+                                                                        Height = 50,
+                                                                        Width = 0.9f,
+                                                                        Text = "Refresh",
+                                                                        Action = refresh
+                                                                    },
+                                                                    new TriangleButton
+                                                                    {
+                                                                        Text = "Import",
+                                                                        Anchor = Anchor.BottomCentre,
+                                                                        Origin = Anchor.BottomCentre,
+                                                                        RelativeSizeAxes = Axes.X,
+                                                                        Height = 50,
+                                                                        Width = 0.9f,
+                                                                        Action = () =>
+                                                                        {
+                                                                            var d = currentFile.Value?.FullName;
+                                                                            if (d != null)
+                                                                                startImport(d);
+                                                                            else
+                                                                                currentFileText.FlashColour(Color4.Red, 500);
+                                                                        },
+                                                                    }
+                                                                },
+                                                            }
+                                                        },
+                                                    }
+                                                },
+                                            }
+                                        }
+                                    }
+                                }
+                            },
+                        }
+                    }
+                }
+            };
```

Comment:
this just can't exist. you probably want to split some of this out into its own component or re-think structure in general.